source/systemadministration/install-on-kvm/kvmserver.rst

```diff
@@ -0,0 +1,290 @@
+Schritt 2: KVM Virtualisierungshost
+====================================
+
+.. hint::
+	Der *kvm* Server bildet das Grundgerüst für die Firewall *ipfire* und
+	den Schulserver *server*. Da KVM im Gegensatz zu Xen oder VMWare auf
+	die Virtualisierungsfunktionen der CPU angewiesen ist, müssen Diese
+	natürlich vorhanden sein.
+
+.. image:: media/kvmserver/kvmserver-image01.png
+
+
+Die folgende Anleitung beschreibt die *einfachste* Implementierung ohne Dinge
+wie VLANs, Teaming oder Raids. Diese Themen werden (hoffentlich schon vorhanden
+;-) in zusätzlichen Anleitungen betrachtet. 
+
+Folgende Hardware sollte der Server mindestens zur Verfügung stellen:
+
+* CPU mit 4 Kernen
+* 16GB RAM
+* 1TB HDD plus zweite HDD für ein Backup
+* 2x 1GBit/s Netzwerkkarten
+
+Schritt 2a: Installation des KVM-Servers
+----------------------------------------
+
+Als Betriebssystem wird *Ubuntu Server* in der Version 16.04 LTS verwendet. Es kann
+`hier <http://releases.ubuntu.com/16.04.3/ubuntu-16.04.3-server-amd64.iso>`_
+heruntergeladen werden.  
+
+**Kopieren des ISOs auf einen USB Stick**
+
+Hilfreiche Befehle sind:
+
+.. code-block:: console
+
+	Löschen des MBRs des USB-Sticks
+	# sudo dd if=/dev/zero of=/dev/sdX bs=1M count=10
+
+	Kopieren des ISOs auf den Stick
+	# sudo dd if=<Name des ISOs> | sudo pv -s <Groesse des ISOs> | sudo dd of=/dev/sdX bs=1M && sync
+
+.. raw:: html
+
+	<p>
+	<iframe width="696" height="392" src="https://www.youtube.com/embed/7NIoQpSSVQw?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+	</p>
+..
+
+**Installation**
+
+.. hint::
+	Bei der Installation sind folgende Merkmale zu berücksichtigen:
+
+		* Auswahl des HWE Kernels
+		* Einrichtung eines LVMs auf der HDD mit 25GB für das Betriebssystem
+		* Auswahl der Pakete *Virtual Machine host* und *OpenSSH server*
+
+	Des Weiteren ist es sinnvoll, die erste Netzwerkkarte des Servers an
+	den Internet-Router anzuschließen, um eventuell notwengige Pakete
+	(Sprachpakete) während der Installation zu installieren.	
+
+.. raw:: html
+
+	<p>
+	<iframe width="696" height="392" src="https://www.youtube.com/embed/ZL0e07nJI_w?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+	</p>
+..
+ 
+Schritt 2b: Netzwerkkonfiguration des KVM-Servers
+-------------------------------------------------
+
+In diesem Schritt erfolgt die Anbindung des KVM-Servers an das Schulnetz und an das Internet sowohl für den KVM-Server selbst, als auch für die virtuellen Maschinen. Die Implementierung erfolgt mit Hilfe von Bridges.
+
+**Herausfinden der Namen der Netzwerkkarten**
+
+.. code-block:: console
+
+	# dmesg | grep eth
+
+**Anpassen der Netzwerkkonfiguration**
+
+.. code-block:: console
+
+	# vi /etc/network/interfaces
```

Comment:
Auch hier würde ich entweder einen anderen Editor nehmen oder einen Hinweis zu "Wie komme ich aus vi wieder raus" dazu :) Im Video sieht man zwar ":wq" aber nicht, wie man in den Einfügenmodus kommt bzw. wieder raus, da Tasenkombinationen nicht angezeigt werden.

Reply:
Kann auf eine vi Grundlagendoku verweisen ...

Reply:
Ich würde es so einfach wie möglich machen. Ich vermute, dass die wenigsten Leser der Doku `vi` verwenden. In den meisten Online-HoTows wird `nano` verwendet - ich denke, genau aus diesem Grund. Nichts gegen `vi`, aber es hebt die Hürde für Einsteiger unnötig.